### ISSUE TITLE
updateIndices.js script broken by Babel 6 upgrade

### ISSUE THREAD

**Issue by wincent** [wincent/masochist#43]
I just updated the "Hammerspoon" article. You can see it remaining in its original position within the list, even though the modification date updated:

<img width="1069" alt="wiki_ _wincent_com" src="https://cloud.githubusercontent.com/assets/7074/11671790/8a55ab12-9dbf-11e5-8c9f-bd5116452402.png">

Push output looked normal as far as indexing goes:

```
remote: 11:19:42 PM: Preparing timestamp cache for revision d7a4abd7edd66e4a42fc151ad87abed5351bbca9
remote: 11:19:42 PM: Preparing list of file updates.
remote: 11:19:42 PM: Getting wiki changes in range: d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote: 11:19:42 PM: Getting snippets changes in range: d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote: 11:19:42 PM: Getting blog changes in range: d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote:
remote: 11:19:42 PM: Writing timestamp cache for revision d7a4abd7edd66e4a42fc151ad87abed5351bbca9
remote: 11:19:42 PM: Creating ordered index
remote: 11:19:42 PM: Preparing list of file updates.
remote: 11:19:42 PM: Getting wiki changes in range: 21b10709d67e4ce4fd7a241bb854a8a128e1b2bb..d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote: 11:19:42 PM: Getting snippets changes in range: 21b10709d67e4ce4fd7a241bb854a8a128e1b2bb..d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote: 11:19:42 PM: Getting blog changes in range: 21b10709d67e4ce4fd7a241bb854a8a128e1b2bb..d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote:
remote: 11:19:42 PM: Creating tag sets
remote: 11:19:42 PM: Preparing list of file updates.
remote: 11:19:42 PM: Getting wiki changes in range: 21b10709d67e4ce4fd7a241bb854a8a128e1b2bb..d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote: 11:19:42 PM: Getting snippets changes in range: 21b10709d67e4ce4fd7a241bb854a8a128e1b2bb..d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote: 11:19:42 PM: Getting blog changes in range: 21b10709d67e4ce4fd7a241bb854a8a128e1b2bb..d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
remote:
remote: 11:19:42 PM: Sending index updates to Redis.
remote: 11:19:42 PM: Finished updating index for revision d7a4abd7edd66e4a42fc151ad87abed5351bbca9.
```

**Comment by wincent** [comment]
The title is a bit of a misnomer, so I'll edit this. The `updateIndices` script is busted as a result of the Babel 6 upgrade. Specifically, the Git commands aren't being awaited, so we don't get any commits. Noticed this when I invalidated the cache and it didn't rebuild (deleted the index during trouble-shooting, which made the site look empty, so I am rebuilding the index now using the rev just prior to the Babel update).

**Comment by wincent** [comment]
Think it's fixed now, via b4a0112 (and later reworked as e38ca73).
